packages/mui-lab/src/Masonry/Masonry.js

```diff
@@ -324,7 +335,14 @@ const Masonry = React.forwardRef(function Masonry(inProps, ref) {
       ownerState={ownerState}
       {...other}
     >
-      {children}
+      {React.Children.map(children, (child) => {
+        return React.cloneElement(child, {
+          style: {
+            ...child.props.style,
+            flexShrink: 0,
+          },
+        });
+      })}
```

Comment:
Would it be better to do this with the pseudo selector `& > *`?

```js
'& > *': {
  flexShrink: 0,
}
```

If we want to force the flex-shrink, it's sounds like we can use `!important`.

Reply:
Definitely better! Refactored 😊